src/com/projectkorra/projectkorra/BendingManager.java

```diff
@@ -1,7 +1,6 @@
 package com.projectkorra.projectkorra;
 
 import java.util.HashMap;
-import java.util.Iterator;

```

Comment:
Just curious, did you do a check through all the files for unused imports and this was the only one?

Reply:
He probably didn't do it for all files but this is certainly an unused import

Reply:
I hope it's unused haha. IntelliJ will tell you where all of them are, figure he may as well get rid of all of them.